### DISCUSSION TITLE
Option to disable UI changes

### DISCUSSION THREAD

**Discussion by VANG06** [burkybang/SuiteAdvanced-Feedback#204] [Ideas]
I want to disable the UI changes that SuiteAdvanced makes, such as overlaid buttons and changes to menu spacing etc, this would allow use with the new redwood theme, without losing the most important features like searching fields etc.

**Comment by HunterJacobs** [comment]
Yes, I would love a enable/disable for UI. I run into some very minute issues with text formatting especially in the dark theme and it would be amazing to quickly disable for a page to see what is needed then re-enable. That way I wouldn't have to log out and in to Suite advanced every time.

**Comment by burkybang** [comment]
There's already a way to quickly disable SuiteAdvanced.

![image](https://github.com/user-attachments/assets/f7a5baa7-aaed-43a2-a68e-d8919b5e3b0f)

**Reply by atrent0815** [reply]
I agree that there should be a way to specifically disable changes to the visual interface. We are finding that SuiteAdvanced is negatively impacting page loading times (only improves after frequently clearing cached images and files in Chrome). We don't want to disable all the features of SuiteAdvanced in the account (for example, we would still like the ability to use live view, click on field help to get to field editing) but we do not need all of the interface shortcuts (for example, changing menu sizes, the format of multi-select fields)

**Reply by atrent0815** [reply]
@burkybang Could you add keyboard shortcuts to enable/disable SuiteAdvanced?

**Reply by burkybang** [reply]
Can you suggest optimal ones?

**Reply by atrent0815** [reply]
I don't have a preference but perhaps CTRL+SHIFT+E for enable/disable for this tab and CTRL+SHIFT+S for enable/disable for the account?

**Reply by atrent0815** [reply]
@burkybang now that I reflect on it more, in addition to keyboard shortcuts, perhaps a more useful feature is the ability to disable the extension in specific roles. I tend to want it running in the admin role but then when I switch to a user role, I turn it off so I can create documentation/demo in the standard interface.

**Reply by theodditie** [reply]
Thank you @burkybang I actually came here to see if there was a way to disable it because I wasn't finding it, but I wasn't hovering over the icon either. Working on some new forms and some of the field display settings don't really impact what I see with the SuiteAdvanced enabled.

@atrent0815 I'll have to remember that when making documentation and demos on processes. Majority of the time I'll pop out of Dark mode when walking through a process, but that's it. I really should get in the habit of just disabling it all together so I know it's the same they see.

I have also noticed the delays on page loading times because of the extension. I ended up disabling the "Indicate Submenu" option and that alone helped clear things up for a bit longer.

**Reply by burkybang** [reply]
> some of the field display settings don't really impact what I see with the SuiteAdvanced enabled

@theodditie Could you create a [New Issue](https://github.com/burkybang/SuiteAdvanced-Feedback/issues/new?template=bug_report.md) about what you're experiencing?

**Reply by theodditie** [reply]
@burkybang I don't think it's worth calling it an issue. Having the ability to disable it on the page or tab to check the display is fine for me.

**Reply by burkybang** [reply]
@theodditie It sounds like you were mentioning a problem. I just want to fix it.
